docs/docs/jobs.md

```diff
@@ -121,6 +121,7 @@ Some interesting job attributes include:
 * `job.started_at`
 * `job.ended_at`
 * `job.exc_info` stores exception information if job doesn't finish successfully.
+* `job.get_heartbeat()` returns last heartbeat of the job indicating last time the job was executing. Can be used to determine if a worker was killed forcely and to mark the job as `failed`.
```

Comment:
This needs to be updated to `job.last_heartbeat` - the latest timestamp that's periodically updated when the job is executing. Can be used to determine if the job is still active.

Reply:
done